app/views/flood_risk_engine/enrollments/steps/_local_authority_address.html.erb

```diff
@@ -1 +1,34 @@
-local_authority_address
+<%= t('.postcode_hint') %>
+<p>
+  <%= form.post_code %>
+  <%= link_to t('.change'),
+              enrollment_step_path(form.enrollment, form.enrollment.previous_step),
+              class: 'change-postcode-button'
+  %>
+</p>
+
+<%= f.hidden_field :post_code, value: form.post_code %>
+
+<label class="form-label" for="address_match_selection">
+  <%= t('.select_from_list_label') %>
+</label>
+
+<% form.address_list.unshift({ moniker: t('.num_addresses_found', num:  form.address_list.count) }) %>
+
+<fieldset>
+  <%= form_group_and_validation(form, :uprn) do %>
+    <%= f.select :uprn,  options_for_select( form.address_list.collect { |a| [ a[:moniker], a[:uprn] ] }), {},
+                 { include_blank: false,
```

Comment:
it didn't work all the time -  when Back or back link used for some reason it disappeared. lots of comments on apidock saying issues with prompt, so maybe that's why wex coded it this way  in first place .. hence the comment
`+<%# Create prompt manually cos this too flakey to use - e.g disappears on Back`